packages/cozy-harvest-lib/src/components/cards/AppLinkCard.jsx

```diff
@@ -23,7 +24,13 @@ const AppLinkCard = ({
   t
 }) => {
   const cozyURL = new URL(client.getStackClient().uri)
-  const { cozySubdomainType: subDomainType } = client.getInstanceOptions()
+  const { fetchStatus, url, isInstalled } = useAppLinkWithStoreFallback(
+    slug,
+    client,
+    path
+  )
+
+  if (fetchStatus !== 'loaded') return null
```

Comment:
Oh again a new case. 

Should we display a busy button when we wait the query's return?

Reply:
Yeah this one is intentional. The other buttons look similar whether or not the app is already installed, but here the label and icons change. I figured this would make the button change too much once the data is loaded, so I thought it would be better to only display it once we know what it should look like.